frontend/sw360-portlet/src/main/resources/META-INF/resources/html/projects/includes/projects/obligationsEdit.jspf

```diff
@@ -383,9 +403,20 @@ require(['jquery', 'bridges/datatables', 'utils/render', 'modules/button', 'modu
     $("#editObligationsTable tbody").on("change", 'td input.obl_comment', function () {
         if ($(this).closest('tr').find("input.obl_comment").val()) {
             storeData($(this));
+            saveStatusAndObl($(this));
         }
     });
 
+    function saveStatusAndObl(thisObj) {
```

Comment:
please change the method name to saveStatusAndComment as this method is used to save these 2 fields of obligation.

Reply:
Hi, I have made the changes. Please have a look.